src/rocksdb.rs

```diff
@@ -513,6 +513,81 @@ impl DB {
         self.get_cf_opt(cf, key, &ReadOptions::new())
     }
 
+    pub fn multi_get_cf_opt(&self,
+                            cf: &CFHandle,
+                            keys_vec: Vec<&[u8]>,
+                            readopts: &ReadOptions)
+                            -> Result<Option<Vec<DBVector>>, String> {
```

Comment:
should be `Result<Vec<Option<DBVector>>, _>`
or more precisely
``Vec<Result<Option<DBVector>, String>>``

since we use `Option<>` to denote Status::NotFound